pages/priorities/index.html

```diff
@@ -0,0 +1,556 @@
+{% assign platforms_priorities = site.priorities | where: "category", "platforms" | sort: "rank" %}
+{% assign android_priorities = site.priorities | where: "category", "android" | sort: "rank" %}
+{% assign linuxbsd_priorities = site.priorities | where: "category", "linuxbsd" | sort: "rank" %}
+{% assign macos_priorities = site.priorities | where: "category", "macos" | sort: "rank" %}
+{% assign web_priorities = site.priorities | where: "category", "web" | sort: "rank" %}
+{% assign xr_priorities = site.priorities | where: "category", "xr" | sort: "rank" %}
+{% assign scripting_priorities = site.priorities | where: "category", "scripting" | sort: "rank" %}
+{% assign gdscript_priorities = site.priorities | where: "category", "gdscript" | sort: "rank" %}
+{% assign dotnet_priorities = site.priorities | where: "category", "dotnet" | sort: "rank" %}
+{% assign gdextension_priorities = site.priorities | where: "category", "gdextension" | sort: "rank" %}
+
+{% include header.html %}
+
+<!-- Priorities stylesheet. -->
+<link rel="stylesheet" href="/assets/css/priorities/base.css" />
+<noscript>
+	<link rel="stylesheet" href="/assets/css/priorities/noscript.css" />
+</noscript>
+
+<!-- Priorities script. -->
+<script type="module" src="/assets/js/priorities.mjs"></script>
+
+<!-- Google icons. -->
+{% assign google_font_icons = page.google_font_icons | sort | join: "," %}
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200&icon_names={{ google_font_icons }}" />
+<style>
+	.material-symbols-outlined {
+	  font-variation-settings:
+	  'FILL' 0,
+	  'wght' 400,
+	  'GRAD' 0,
+	  'opsz' 24
+	}
+</style>
+
+<div class="priorities-container">
+	<div class="banner">
+		<div class="container">
+			<h1>Priorities</h1>
+			<div class="banner-last-updated">
+				{% assign is_recent = false %}
+				{% assign last_item_updated = site.priorities | sort: "last_updated" | last %}
+				{% assign last_updated = last_item_updated.last_updated %}
+				{% if last_updated != "" %}
+					{% assign last_updated_time = last_updated | date: "%s" | plus: 0 %}
+					{% assign last_updated_target = now | minus: last_updated_duration %}
+					{% if last_updated_time > last_updated_target %}
+						{% assign is_recent = true %}
+					{% endif %}
+				{% endif %}
+				<div class="label">
+					{% if is_recent and enable_recent_update %}
+						<span class="recently-updated">Recently updated!</span>
+					{% else %}
+						<span class="last-updated">Last updated</span>
+					{% endif %}
+				</div>
+				<div class="date">
+					<time datetime="{{ last_updated }}">{{ last_updated }}</time>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div class="intro">
+		<div class="container">
+			<p class="wishlist-par">
+				This page can be understood as a <strong>wishlist</strong>:
+				it details the software improvements that we deem necessary to make the Godot Engine a better tool for its users.
```

Comment:
```suggestion
				it details the software improvements that we deem high priority to make the Godot Engine a better tool for its users.
```

I don't want to nitpick but this stood out to me when reading the draft - if it's the list of *necessary* improvements, why is it a *wishlist*? We might want to soften this a little. The rest of the copy already does, but it comes after this very strong "necessary" phrasing early on.

Reply:
@clayjohn What do you think?

Reply:
```suggestion
				it details the software improvements that we consider most important to make the Godot Engine a better tool for its users.
```

Here's a wording that doesn't imply either a guarantee that a feature *will* be implemented nor that it will be worked on *soon*

Reply:
I used your latest wording, @tetrapod00 !